source/sync/data-access-patterns/sync-mode.txt

```diff
@@ -36,7 +36,7 @@ Those documents form a **partition**.
 In Partition-Based Sync, you define a :ref:`partition key <partition-key>` 
 whose value determines whether the user can read or write a document. 
 {+service+} :ref:`evaluates a set of rules <partition-based-rules-and-permissions>` 
```

Comment:
Would we say that "Atlas App Services" evaluates a set of rules, or would it be "Device Sync?" I'm uncertain how to communicate around this (and it's mentioned a few times on the page, and probably on the configuration page.)